huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_port_test.go

```diff
@@ -31,6 +35,34 @@ func TestAccIECPortDataSource_basic(t *testing.T) {
 	})
 }
 
+func testAccIECNetworkConfig_base(rName string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_iec_sites" "sites_test" {}
+
+resource "huaweicloud_iec_vpc" "vpc_test" {
+  name = "%[1]s"
+  cidr = "192.168.0.0/16"
+  mode = "CUSTOMER"
+}
+
+resource "huaweicloud_iec_vpc_subnet" "subnet_1" {
+  name       = "%[1]s-1"
+  cidr       = "192.168.0.0/24"
+  vpc_id     = huaweicloud_iec_vpc.vpc_test.id
+  site_id    = data.huaweicloud_iec_sites.sites_test.sites[0].id
+  gateway_ip = "192.168.0.1"
+}
+
+resource "huaweicloud_iec_vpc_subnet" "subnet_2" {
+  name       = "%[1]s-2"
+  cidr       = "192.168.1.0/24"
+  vpc_id     = huaweicloud_iec_vpc.vpc_test.id
+  site_id    = data.huaweicloud_iec_sites.sites_test.sites[1].id
+  gateway_ip = "192.168.1.1"
+}
```

Comment:
```suggestion
resource "huaweicloud_iec_vpc_subnet" "test" {
  count = 2

  name       = format("%[1]s-%%d", count.index)
  cidr       = cidrsubnet(huaweicloud_iec_vpc.vpc_test.cidr, 4, count.index)
  vpc_id     = huaweicloud_iec_vpc.vpc_test.id
  site_id    = try(data.huaweicloud_iec_sites.sites_test.sites[count.index].id, "")
  gateway_ip = cidrhost(cidrsubnet(huaweicloud_iec_vpc.vpc_test.cidr, 4, count.index), 1)
}
```
this function seem the function of the IEC vpc subnet test.
support a new one?